sideboard/internal/connection_checker.py

```diff
@@ -48,12 +48,12 @@ def _check(url, **ssl_params):
 
         try:
             with closing(socket.create_connection((host, port))) as sock:
-                wrapped = ssl.wrap_socket(sock, cert_reqs=ssl.CERT_REQUIRED, **ssl_params)
+                wrapped = ssl.wrap_socket(sock, **dict(ssl_params, cert_reqs=ssl.CERT_REQUIRED))
+                status.append('succeeded at validating server cert')
```

Comment:
MAGFest doesn't make much use of our server API, but this adds an extra check to our automatic SSL checking.